Packs/RiskSense/Playbooks/playbook-CVE_Exposure_-_RiskSense.yml

```diff
@@ -0,0 +1,547 @@
+          - operator: Stringify
+          - operator: join
+            args:
+              separator:
+                value:
+                  simple: ','
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 1215
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "17":
+    id: "17"
+    taskid: b31a916a-9752-45fb-8354-5856a5124be6
+    type: playbook
+    task:
+      id: b31a916a-9752-45fb-8354-5856a5124be6
+      version: -1
+      name: Block IP - Generic v2
+      playbookName: Block IP - Generic v2
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      AutoCommit:
+        simple: "No"
+      CustomBlockRule:
+        simple: "True"
+      DAG: {}
+      EDLServerIP: {}
+      IP:
+        complex:
+          root: RiskSense
+          accessor: HostFinding.HostIpAddress
+          transformers:
+          - operator: uniq
+      IPBlacklistMiner: {}
+      IPListName: {}
+      LogForwarding: {}
+      StaticAddressGroup: {}
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 910,
+          "y": 1215
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "18":
+    id: "18"
+    taskid: 7a49df8d-d2c9-44a4-897a-4ea385ed52f3
+    type: condition
+    task:
+      id: 7a49df8d-d2c9-44a4-897a-4ea385ed52f3
+      version: -1
+      name: Are Inputs provided?
+      description: Returns 'yes' if inputs are provided. Otherwise returns 'no'
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "10"
+      "yes":
+      - "1"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.CVES
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: inputs.TagName
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 130,
+          "y": 175
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: |-
+  {
+    "linkLabelsPosition": {
+      "12_10_#default#": 0.17,
+      "12_15_yes": 0.76,
+      "18_1_yes": 0.18,
+      "1_10_#default#": 0.8
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1535,
+        "width": 1510,
+        "x": -220,
+        "y": 40
+      }
+    }
+  }
+inputs:
+- key: CVES
+  value:
+    simple: CVE-2017-11882,CVE-2017-0199,CVE-2017-5638,CVE-2012-0158
```

Comment:
Were these CVE's left here intentionally? better to not hard code this if this is just an example.

Reply:
@David-BMS this was missed, removing default values.